### PR TITLE
Add dev container info to "How to Contribute"

### PR DESCRIPTION
**Pull request by bamurtaugh**
While we have guidance in using the dev container on the [main readme](https://github.com/microsoft/vscode#development-container), a user was searching for it in the "How to Contribute" wiki, and it seems like it'd make sense to include it there as well.

See https://github.com/microsoft/vscode/issues/103293

### CODE REVIEW

**Comment by Chuxel** [comment]
If they're looking for contributing list of dev containers in the  the Remote - Containers extension, we can also route them to the CONTRIBUTING.md in the vscode-dev-containers repo.  I've tried to keep it up to date over time.

https://github.com/microsoft/vscode-dev-containers/blob/master/CONTRIBUTING.md

**Comment by bamurtaugh** [comment]
Since we're referencing dev containers in the "How to Contribute" to VS Code wiki at large, were you thinking it might make sense to add how to contribute to the dev container list in the Remote - Containers extension at large in this new section of the wiki? That's an interesting idea, I hadn't thought of it.

Were you thinking something like: 

`If you'd like to contribute to the list of available development containers in the Remote - Containers extension, you can check out the [Contributing documentation](https://github.com/microsoft/vscode-dev-containers/blob/master/CONTRIBUTING.md) in the vscode-dev-containers repo.`

**Comment by Chuxel** [comment]
Yep! That works.

File: How-to-Contribute.md [hunk 1]
```diff
@@ -64,6 +64,17 @@ In case of issues, try deleting the contents of `~/.node-gyp` (alternatively `~/
...
 > If you have Visual Studio 2019 installed, you may face issues when using the default version of node-gyp. If you have Visual Studio 2019 installed, you may need to follow the solutions [here](https://github.com/nodejs/node-gyp/issues/1747)
 
+### Development container
+
+Alternatively, you can avoid local dependency installation as this repository includes a Visual Studio Code Remote - Containers / Codespaces [development container](https://github.com/microsoft/vscode/tree/master/.devcontainer).
+
+- For [Remote - Containers](https://aka.ms/vscode-remote/download/containers), use the **Remote-Containers: Open Repository in Container...** command which creates a Docker volume for better disk I/O on macOS and Windows.
+- For Codespaces, install the [Visual Studio Codespaces](https://aka.ms/vscs-ext-vscode) extension in VS Code, and use the **Codespaces: Create New Codespace** command.
+
+Docker / the Codespace should have at least **4 Cores and 6 GB of RAM (8 GB recommended)** to run full build. See the [development container README](https://github.com/microsoft/vscode/blob/master/.devcontainer/README.md) for more information.
```
Comment (chrmarti) [inline How-to-Contribute.md:74]: run full build > run the full build

Comment (bamurtaugh) [reply How-to-Contribute.md]: Good catch, added!

**Pull request merged**